source/MultiGrid/MultiGrid.js

```diff
@@ -366,6 +369,8 @@ export default class MultiGrid extends Component {
         left: 0,
         outline: 0,
         overflow: 'hidden',
+        overflowX: 'hidden',
+        overflowY: 'hidden',
```

Comment:
I think they are needed to counteract gridStyle.overflowX and gridStyle.overflowY being set in
https://github.com/bvaughn/react-virtualized/blob/master/source/Grid/Grid.js#L566